From c68c72144d7b2a0516276137855d4508a3f59a17 Mon Sep 17 00:00:00 2001
From: Michael Hutchinson <m.j.hutchinson@gmail.com>
Date: Tue, 1 Apr 2014 17:21:51 -0400
Subject: [PATCH] [xbuild] Include PCL common targets in tarball

BXC18690 - Microsoft.Portable.Common.targets is missing in mono 3.4.0 tarball
---
 mcs/tools/xbuild/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mcs/tools/xbuild/Makefile b/mcs/tools/xbuild/Makefile
index c3a5d6a..31d5329 100644
--- a/mcs/tools/xbuild/Makefile
+++ b/mcs/tools/xbuild/Makefile
@@ -93,6 +93,7 @@ EXTRA_DISTFILES = \
 	frameworks/net_4.5.xml \
 	targets/Microsoft.Portable.CSharp_4.0.targets \
 	targets/Microsoft.Portable.CSharp_4.5.targets \
+	targets/Microsoft.Portable.Common.targets \
 	targets/Microsoft.Portable.Core.targets \
 	targets/Microsoft.Portable.Core.props \
 	targets/Microsoft.WebApplication.targets \
-- 
1.9.3

